examples/swash_render/src/main.rs

```diff
@@ -141,7 +141,9 @@ fn main() {
 
         // Set default font family
         builder.push_default(font_stack);
-        builder.push_default(StyleProperty::LineHeight(1.3));
+        builder.push_default(StyleProperty::LineHeight(parley::LineHeight::SizeRelative(
+            1.3,
+        )));
```

Comment:
Could we add an `Into` from `LineHeight` to `StyleProperty::LineHeight` to make this a little less verbose?

Reply:
I think that impl exists, I just did a quick search-and-replace over the examples and forgot to update them to actually use it.